ui/admin/tests/acceptance/aliases/create-test.js

```diff
@@ -56,37 +49,37 @@ module('Acceptance | aliases | create', function (hooks) {
     const aliasCount = getAliasCount();
     await visit(urls.newAlias);
 
-    await fillIn(NAME_FIELD_SELECTOR, NAME_FIELD_TEXT);
-    await fillIn(DESTINATION_ID_SELECTOR, 'tcp_123');
-    await fillIn(HOST_ID_SELECTOR, 'h_123');
-    await click(SAVE_BTN_SELECTOR);
+    await fillIn(commonSelectors.FIELD_NAME, commonSelectors.FIELD_NAME_VALUE);
+    await fillIn(selectors.DESTINATION_ID_SELECTOR, 'tcp_123');
```

Comment:
If you feel the value we are fillingIn this field will be need again `tcp_123`, in example, to validate an assertion when updating the field, the pattern we follow is to create a const for it. For example a [field selector + field value on auth-methods domain](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/acceptance/auth-methods/selectors.js#L10-L11) or [accounts selectors](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/acceptance/accounts/selectors.js#L24-L25).
If you know or no need now to repeat this value, hardcoding is totally fine 😉

Reply:
I verified in the aliases acceptance tests (create, read, update, list, delete) that those two fields are never reused. 👍